db/migrations/20240924194257_add_regression_test_version_to_service_deployments.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.alterTable('service_deployments', (t) => {
+    t.string('regression_test_version');
```

Comment:
nit: maybe use `t.string('regression_test_version', 255);` to use a `varchar(255)` here?